### ISSUE TITLE
document Conventional Commits and how they inform releases with auto

### ISSUE THREAD

**Issue by avkvirtru** [avkvirtru/virtuoso-design-system#58]
# 📚 Docs issue report

### Description

@jrschumacher and I decided to try [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) for this repo. The commit linter enforces this, but there should ideally be a section in the README to describe what commits the repo expects.

Sample discussion from Slack:

> 10:43 AM
> Art Klepchukov I mean new components are features right, which mean minor releases (edited) 
> 10:43 AM
> most things are fixes, which are patches
> 10:44 AM
> and occasional breaking things will be what they’ll be
> 10:44 AM
> i don’t mind being hands off
> 10:44 AM
> lefthook encourages better commits
> 10:44 AM
> Ryan Schumacher :spiral_calendar_pad: I just mean we need to make sure people are using the right commit messages.
> 10:44 AM
> Art Klepchukov better commits encourage better semver adherence
> 10:44 AM
> and auto takes it from there

**Comment by jrschumacher** [comment]
Agreed. I did update the git commit-msg hook to provide more details.

```
-------------------------------------------------------------------------------------------
This project uses conventional commits, please make sure your commit message looks like:

    type(scope?): subject

i.e.

    chore: run tests on github
    fix(button): pass event handler correctly
    feat(table): add filter for columns

Learn more at https://www.conventionalcommits.org.
-------------------------------------------------------------------------------------------
⧗   input: blah
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```
